Sources/Swift/Protocol/SentryLogLevel.swift

```diff
@@ -0,0 +1,28 @@
+extension SentryLog {
+    enum Level: String, Codable {
+        case trace
+        case debug
+        case info
+        case warn
+        case error
+        case fatal
+        
+        // Docs: https://develop.sentry.dev/sdk/telemetry/logs/#log-severity-number
+        func toSeverityNumber() -> Int {
```

Comment:
`m`: Codecov rightfully complains about missing code coverage. Please add some tests validating this functionality.